src/coreclr/jit/emit.h

```diff
@@ -763,8 +772,9 @@ class emitter
 
         unsigned _idCnsReloc : 1; // LargeCns is an RVA and needs reloc tag
         unsigned _idDspReloc : 1; // LargeDsp is an RVA and needs reloc tag
+        unsigned _idIsJmpAlways : 1;
```

Comment:
Do we really need a bit here? Can't we just check the instruction?

Reply:
It isn't strictly required but it limits the scope of work that needs to be done. In complex functions the number of jumps can get quite high and iterating the ig for all of them is mostly a waste of time This will limit the throughput loss. In general the number of removed jumps is 1-3 per function from the diffs that I've checked.

Reply:
Another alternative would be to create a `emitUncondJumpList` (similar to `emitJumpList`) which just keeps track of unconditional jumps. You will then just eliminate `emitIsUncondJump(jmp) && jmp->idGetJmpAlwaysFlag()` and won't have to go through all the other types of jumps.

Reply:
I considered this but when I reviewed the logic in the emitter around saving the current ig I decided I didn't really want to disturb it.

Reply:
I think tracking a dedicated list might be cleaner and better TP wise.

Reply:
Moved to instrDesJmp as suggested in later review because the dedicated list implementation added more complexity than it removed.